crates/jstzd/tests/jstzd_test.rs

```diff
@@ -22,13 +23,41 @@ const CONTRACT_NAMES: [(&str, &str); 2] = [
 #[tokio::test(flavor = "multi_thread")]
 async fn jstzd_test() {
     let rpc_endpoint = Endpoint::localhost(unused_port());
+    let jstzd_port = unused_port();
+    let (mut jstzd, config) = create_jstzd_server(&rpc_endpoint, jstzd_port).await;
+
+    jstzd.run().await.unwrap();
+    ensure_jstzd_components_are_up(&rpc_endpoint, jstzd_port).await;
+    let octez_client = OctezClient::new(config.octez_client_config().clone());
+    check_bootstrap_contracts(&octez_client).await;
+
+    assert!(jstzd.health_check().await);
+
+    fetch_config_test(config, jstzd_port).await;
+
+    reqwest::Client::new()
+        .put(&format!("http://localhost:{}/shutdown", jstzd_port))
+        .send()
+        .await
+        .unwrap();
+
+    ensure_jstzd_components_are_down(&rpc_endpoint, jstzd_port).await;
+
+    // stop should be idempotent and thus should be okay after jstzd is already stopped
+    jstzd.stop().await.unwrap();
```

Comment:
Yes. I want to test `.stop()` but at the same time I don't feel like running a separate test just for that. I've removed this line since it does not do much here.